src/main/kotlin/no/nav/familie/ks/sak/api/BehandlingStegController.kt

```diff
@@ -51,4 +51,15 @@ class BehandlingStegController(
 
         return ResponseEntity.ok(Ressurs.success(behandlingService.lagBehandlingRespons(behandlingId = behandlingId)))
     }
+
+    @PostMapping(path = ["behandlingsresultat"])
+    fun utledBehandlingsresultat(@PathVariable behandlingId: Long): ResponseEntity<Ressurs<BehandlingResponsDto>> {
+        tilgangService.validerTilgangTilHandling(
+            minimumBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
+            handling = "vurdere behandlingsresultat"
```

Comment:
```suggestion
            handling = "utled behandlingsresultat"
```
```suggestion
            handling = "utfør vilkårsvurdering"
```
1-1 handling og metodenavn